src/SearchBundle/Bridge/ShopSearchSessionChameleonBridge.php

```diff
@@ -69,4 +77,15 @@ private function getArrayAsString($data)
 
         return implode(',', $parts);
     }
+
+    private function getSession(): ?SessionInterface
+    {
+        $request = $this->requestStack->getCurrentRequest();
+        
+        if (null === $request) {
+            return null;
+        }
+                
```

Comment:
If the session does not exist, an exception is thrown. To avoid this, check the exististence of it before.
(See also the usual `getSession`-Deprecation warning)
```suggestion
        if (null === $request) {
            return null;
        }
        
        if (false === $request->hasSession()) {
            return null;
        }

```